app/uk/gov/hmrc/mtdsatestsupportapi/controllers/requestParsers/validators/validations/BusinessIncome2YearsPriorValidation.scala

```diff
@@ -30,7 +30,7 @@ object BusinessIncome2YearsPriorValidation {
 
   private def validate(field: BigDecimal, path: String): List[MtdError] = {
     if (field >= 0 && field < 99999999999.99 && field.scale <= 2) NoValidationErrors
```

Comment:
need to update the max field to field <= 99999999999.99 so it includes 99999999999.99 as a value too